src/hipscat/pixel_math/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Include all modules in this subdirectory"""
+
+from .partition_stats import *
```

Comment:
Don't do this `*` import, this would litter the namespace with healpy, numpy, etc, too as opposed to picking up only the public functions and classes from the module. Either:

- be specific and import only what is supposed to end up in the namespace
- define `__all__` in the module, listing the public facing functionality, so the `*` import would pick up only what's relevant. 

This comment may apply to other places in this PR, too, but I don't repeat it.

Reply:
I think I've fixed this throughout the library. Thanks!